app/queries/scorebook.rb

```diff
@@ -50,6 +50,7 @@ def student_activity_sessions(classroom_id)
     # Find all the 'final' activity sessions for all the students in all the classrooms
     results = ActivitySession.select("users.name, activity_sessions.id, activity_sessions.percentage,
                                 #{User.sorting_name_sql}")
+                              .preload(:concept_results => :concept)
```

Comment:
@ddmck looks like preload was the trick - I think we were just adding it on the other side of includes, so apparently it likes preload before includes